jenkins.py

```diff
@@ -106,7 +106,16 @@ def _api_call(url, opener, http_timeout):
     try:
         urllib2.install_opener(opener)
         resp = urllib2.urlopen(url, timeout=http_timeout)
-    except (urllib2.HTTPError, urllib2.URLError) as e:
+    except (urllib2.HTTPError) as e:
+	# Workaround for Jenkins issue with health check
+	# which returns 500 server code if a health check fails
+	# https://github.com/sensu-plugins/sensu-plugins-jenkins/issues/10
+	if e.code == 500:
+	    return json.loads(e.read())
```

Comment:
I would not just blindly assume that all 500 responses have valid json.  Wrap the `json.loads` call with another try/except block and just log the standard error message if json parsing fails.